extensions/podman/src/extension.ts

```diff
@@ -76,6 +76,15 @@ const qemuHelper = new QemuHelper();
 const podmanBinaryHelper = new PodmanBinaryLocationHelper();
 const podmanInfoHelper = new PodmanInfoHelper();
 
+let shouldNotifySetup = true;
+const setupPodmanNotification: extensionApi.NotificationInfo = {
+  extensionId: 'podman-desktop.podman',
+  title: 'Podman needs to be setup!',
+  description: 'The Podman extension is installed but needs to be setup. Some features may not work as expected.',
```

Comment:
This is so pedantic 😅 but I would do

s/setup/set up - in both the title and the description. It just comes across more clearly (I think usually "setup" is treated as a noun whereas "set up" is a verb / verbal phrase.)